sys/log/full/src/log.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Licensed to the Apache Software Foundation (ASF) under one
+* Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Accidental whitespace deletion? Might be worth adding back, not sure how rat tool checks these...

Reply:
Huh... not sure how that happened.  Definitely not intentional :).